### PR TITLE
QuoteMedia Component

### PR DESCRIPTION
**Pull request by zeesapient**

### CODE REVIEW

File: lib/components/atoms/Blockquote/Blockquote.story.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+// TODO: indentation is not working fine in editor. Need to check for .hbs files
+
+storiesOf('Atoms/Blockquote', module)
+  .addParameters({
+    jest: ['Blockquote', 'BlockquoteVanilla'],
+  })
+  .add('Blockquote Knobs', () => (
+    <BlockquoteVanilla
+      {...defaultConfig}
+      className={`hide-default-sample ${defaultConfig.className}`}
```
Comment (vinodloha) [inline lib/components/atoms/Blockquote/Blockquote.story.js:18]: This class `hide-default-sample  is no longer needed

File: lib/components/atoms/Blockquote/tests/Blockquote.test.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import Blockquote from '../Blockquote';
```
Comment (vinodloha) [inline lib/components/atoms/Blockquote/tests/Blockquote.test.js]: The test cases should be running on Vanilla components.

File: lib/components/molecules/QuoteMedia/QuoteMedia.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+  title,
+  author,
+  quoteContent,
+  children,
+}: Props): Node => (
+  <div className={className}>
+    <Heading HeadingType="h2" className="title">
+      {title}
+    </Heading>
+    <Blockquote className={childClassName} author={author} quoteContent={quoteContent} />
```
Comment (vinodloha) [inline lib/components/molecules/QuoteMedia/QuoteMedia.js]: I think you need a container here around Blockquote and media that can help you align them properly

File: lib/components/atoms/Blockquote/tests/Blockquote.test.js [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+describe('<Blockquote />', () => {
+  let BlockquoteComponent = '';
+  beforeEach(() => {
+    BlockquoteComponent = shallow(<Blockquote />);
+  });
+  it('should render blockquote', () => {
+    expect(BlockquoteComponent.find('blockquote')).toBeTruthy();
+  });
+  it('should render content', () => {
```
Comment (vinodloha) [inline lib/components/atoms/Blockquote/tests/Blockquote.test.js:18]: Tests could be more detailed, only checking for tag is not enough.

File: lib/components/molecules/QuoteMedia/QuoteMedia.js [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+import type { Node } from 'react';
+import styles from './QuoteMedia.style';
+import type { Props } from './types';
+
+import Heading from '../../atoms/Heading/Heading';
+import Blockquote from '../../atoms/Blockquote/Blockquote';
+
+const QuoteMedia = ({
+  className,
+  childClassName,
```
Comment (vinodloha) [inline lib/components/molecules/QuoteMedia/QuoteMedia.js]: This childClassName naming convention doesn't make it clear where its going to apply. 

quoteContent --> quoteProps

File: lib/components/molecules/QuoteMedia/QuoteMedia.js (hunk 3 above)
Comment (vinodloha) [inline lib/components/molecules/QuoteMedia/QuoteMedia.js]: Using <Blockquote> WithIn this component will invite a change at this component QuoteMedia too. We can remove   this dependency via RenderProp

quoteComponent: () => <Blockquote className={} author={author} quoteContent={quoteContent} />

File: lib/components/molecules/QuoteMedia/QuoteMedia.story.js [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+  .addParameters({
+    jest: ['QuoteMedia', 'QuoteMediaVanilla'],
+  })
+  .add('QuoteMedia Knobs', () => (
+    <QuoteMediaVanilla
+      {...defaultConfig}
+      className={`hide-default-sample ${defaultConfig.className}`}
+    />
+  ))
+  .add('QuoteMedia', () => (
```
Comment (vinodloha) [inline lib/components/molecules/QuoteMedia/QuoteMedia.story.js]: QuoteMedia --> Default

File: lib/components/molecules/QuoteMedia/QuoteMedia.story.js [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+// Import Styled Component to showcase variations
+import QuoteMedia, { QuoteMediaVanilla } from '.';
+
+// TODO: indentation is not working fine in editor. Need to check for .hbs files
+
+storiesOf('Molecules/QuoteMedia', module)
+  .addParameters({
+    jest: ['QuoteMedia', 'QuoteMediaVanilla'],
+  })
+  .add('QuoteMedia Knobs', () => (
```
Comment (vinodloha) [inline lib/components/molecules/QuoteMedia/QuoteMedia.story.js]: QuoteMedia Knobs --> Knobs

File: lib/components/molecules/QuoteMedia/tests/QuoteMedia.test.js [hunk 8]
```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import Blockquote from '../../../atoms/Blockquote/Blockquote';
+import Heading from '../../../atoms/Heading/Heading';
+import QuoteMedia from '../QuoteMedia';
+
+describe('<QuoteMedia />', () => {
+  let QuoteMediaComponent = '';
+  beforeEach(() => {
+    QuoteMediaComponent = shallow(<QuoteMedia />);
```
Comment (vinodloha) [inline lib/components/molecules/QuoteMedia/tests/QuoteMedia.test.js]: This is wrong Test Environment, You should test the component as if its practically going to be used. Give the sample data.

File: lib/components/atoms/Blockquote/Blockquote.style.js [hunk 9]
```diff
@@ -0,0 +1,35 @@
...
+    `};
+  }
+  &:before {
+    display: block;
+    height: 0;
+    content: '”';
+    position: absolute;
+    top: 0;
+    ${props => `
+      ${props.theme.typography.FONT_QUOTE};
```
Comment (vinodloha) [inline lib/components/atoms/Blockquote/Blockquote.style.js]: Rename this to FONT_QUOTE --> FONT_QUOTE_SIGN

File: lib/components/atoms/Blockquote/Blockquote.story.js (hunk 1 above)
Comment (zeesapient) [reply lib/components/atoms/Blockquote/Blockquote.story.js:18]: fixed.

File: lib/components/atoms/Blockquote/tests/Blockquote.test.js (hunk 2 above)
Comment (zeesapient) [reply lib/components/atoms/Blockquote/tests/Blockquote.test.js]: fixed

File: lib/components/molecules/QuoteMedia/QuoteMedia.js (hunk 5 above)
Comment (zeesapient) [reply lib/components/molecules/QuoteMedia/QuoteMedia.js]: fixed

File: lib/components/molecules/QuoteMedia/QuoteMedia.story.js (hunk 6 above)
Comment (zeesapient) [reply lib/components/molecules/QuoteMedia/QuoteMedia.story.js]: fixed

File: lib/components/molecules/QuoteMedia/QuoteMedia.story.js (hunk 7 above)
Comment (zeesapient) [reply lib/components/molecules/QuoteMedia/QuoteMedia.story.js]: fixed

File: lib/components/atoms/Blockquote/Blockquote.style.js (hunk 9 above)
Comment (zeesapient) [reply lib/components/atoms/Blockquote/Blockquote.style.js]: fixed

**Pull request merged**